theme/templates/base.html

```diff
@@ -22,26 +22,47 @@
         {% endif %}
     </head>
     <body class="{% if page and page.body_class %}{{ page.body_class }}{% endif %}">
-        <!-- Nav -->
-        <nav class="col-12 primary">
-            <a href="/">
-                <img src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/images/kiwi_logo_h50.png" name="logo">
-            </a>
-
-            <ul class="topnav">
-                <li class="right">
+        <!-- Mobile Nav -->
+        <aside class="mobilenav primary">
+            <button class="close-menu accent-unimportant"><i class="fa fa-times" aria-hidden="true"></i></button>
+            <ul>
+                <li><a class="active accent" href="https://demo.kiwitcms.org">Checkout Demo</a></li>
```

Comment:
This list is exactly the same as the one below it. We should have only 1 menu and use the above CSS/JS to style and operate on the existing menu. Try to find a solution without duplicating the list of menu links because this list can change at any moment and this duplication will lead to differences (bugs).